transport-native-epoll/src/test/java/io/netty/channel/epoll/EpollSocketTestPermutation.java

```diff
@@ -98,23 +108,40 @@ public ServerBootstrap newInstance() {
         return toReturn;
     }
 
-    @SuppressWarnings("unchecked")
     @Override
     public List<BootstrapFactory<Bootstrap>> clientSocket() {
-        return Arrays.asList(
-                new BootstrapFactory<Bootstrap>() {
-                    @Override
-                    public Bootstrap newInstance() {
-                        return new Bootstrap().group(EPOLL_WORKER_GROUP).channel(EpollSocketChannel.class);
-                    }
-                },
-                new BootstrapFactory<Bootstrap>() {
-                    @Override
-                    public Bootstrap newInstance() {
-                        return new Bootstrap().group(nioWorkerGroup).channel(NioSocketChannel.class);
-                    }
+        return clientSocket(true);
+    }
+
+    @SuppressWarnings("unchecked")
+    public List<BootstrapFactory<Bootstrap>> clientSocket(boolean allowStateless) {
```

Comment:
same comment as above... its not clear what stateless means in this context imho